server/bleep/src/indexes/file.rs

```diff
@@ -532,7 +532,7 @@ impl RepoDir {
         relative_path: &Path,
         repo_disk_path: &Path,
         repo_ref: &str,
-        last_commit: u64,
+        last_commit: i64,
```

Comment:
Why the sign change?

Reply:
It's a dependency requirement, gix changed fixed their apis to track the correct type for unix seconds. (previously it was different from the underlying POSIX APIs for some reason)